modules/hitl/src/backend/db.test.ts

```diff
@@ -0,0 +1,36 @@
+import Db, { TABLE_NAME_SESSIONS, TABLE_NAME_MESSAGES } from './db'
+import 'reflect-metadata'
```

Comment:
what's that for  ?

Reply:
I think I missed this while copying the test file, good catch, I'll delete it